src/NzbDrone.Core/Extras/Subtitles/SubtitleService.cs

```diff
@@ -68,11 +68,18 @@ public override IEnumerable<ExtraFile> MoveFilesAfterRename(Series series, List<
             var subtitleFiles = _subtitleFileService.GetFilesBySeries(series.Id);
 
             var movedFiles = new List<SubtitleFile>();
+            
+            string GetSubtitleFileIdentifier(SubtitleFile subtitleFile)
+            {
+                var languageTags = LanguageParser.ParseLanguageTags(subtitleFile.RelativePath);
+                return subtitleFile.Language + subtitleFile.Extension +
+                       languageTags.Aggregate(string.Empty, (tags, tag) => tags + tag);
+            }
```

Comment:
Maybe it would be better to store the parsed tags in the database than repeatedly parsing them from the file name?

Reply:
Pros and cons to both, if additional parsing is added later reparsing the filename will add more value, but not doing the same thing over and over is also valuable.

Reply:
I've left that as-is, since additional parsing is more likely to be added than not, imo. It will also leave the code "prepared" - minimizing possible future changes size, should someone implement something like that.

However, I'm still open to this and don't mind implementing this the other way.